src/components/GoodsList.tsx

```diff
@@ -0,0 +1,13 @@
+import { Good } from './Good';
+
+type GoddsListProps = {
+  goods: string[]
+};
+
+export const GoodsList = ({ goods }: GoddsListProps) => {
+  return (
+    <ul>
+      {goods.map(good => <Good name={good} key={goods.indexOf(good)} />)}
```

Comment:
[.map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) as the second argument receives index of the currently iterated element, so there is no need for the indexOf
```tsx
{goods.map((good, index) => (
  <Good name={good} key={index} />
))}
```
however you should NEVER use indexes as keys, goods in this case are unique so use the values as keys (also do the implicit return)
```tsx
export const GoodsList = ({ goods }: GoddsListProps) => (
  <ul>
    {goods.map((good) => (
      <Good name={good} key={good} />
    ))}
  </ul>
);
```

Reply:
I deleted return and switched keys to goods.